lib/Onyx.js

```diff
@@ -150,6 +150,45 @@ function addAllSafeEvictionKeysToRecentlyAccessedList() {
         });
 }
 
+/**
+ * When a collection of keys change, search for any callbacks matching the collection key and trigger those callbacks
+ *
+ * @param {String} parentKey
+ * @param {Object} collection
+ */
+function keysChanged(parentKey, collection) {
+    // Add or remove this parentKey from the recentlyAccessedKeys lists
+    if (!_.isNull(collection)) {
+        addLastAccessedKey(parentKey);
+    } else {
+        removeLastAccessedKey(parentKey);
+    }
```

Comment:
I think we don't need these at all since last accessed should never be a collection key and `addLastAccessedKey()` will just return immediately anyway.